app/models/concerns/convertable.rb

```diff
@@ -0,0 +1,9 @@
+module Convertable
+  def format_city_state_zip(data)
+    city = data[:city]
+    state_abbr = data[:state]
+    zip_code = data[:zip_code]
+
+    "#{city}, #{state_abbr} #{zip_code}"
+  end
+end
```

Comment:
Nice use of a module here!  Love it!  I think this follows OOP/SRP better where the module handles formatting instead of the PORO ... because it isn't necessarily the Event's responsibility to have that formatting behavior.

With this logic, I think we can move the `format_date` and `format_date_short` into the module as well.

Perhaps we have the Event PORO store the raw date in the `@date_time` attribute, move these methods into (or `include` the module in) the `ApplicationHelper`, and then do the date formatting in the views?  Thoughts?

If using as a module included in `ApplicationHelper`, we could move the module into the `app/helpers` directory.

Reply:
I agree, I think I'll go with `formatable`, as that's ultimately what both method sets are doing, so it would make sense to group them this way for continuity